README.md

```diff
@@ -17,7 +17,7 @@ npm install angular-message-format
 Then add `ngMessageFormat` as a dependency for your app:
 
 ```javascript
-angular.module('myApp', [require('angular-message-format.min')]);
```

Comment:
I just blindly removed `.min`, but I think you are right. This line won't work.
We should remove it from all modules that have it (not all of them I think, but some do).